### PR TITLE
Merge run-example-tests-ray.sh and automl.sh

### PR DESCRIPTION
**Pull request by sgwhat**

### CODE REVIEW

**Comment by sgwhat** [comment]
Jenkins Passed: http://10.112.231.51:18888/job/ZOO-PR-BigDL-Python-ExampleTests-py37-ray/123/console
http://10.112.231.51:18888/job/ZOO-PR-BigDL-Python-ExampleTests-py36-ray/130/console
http://10.112.231.51:18888/job/ZOO-PR-BigDL-Python-ExampleTests-py37-ray-Spark3/131/console

File: python/orca/dev/example/run-example-test-ray.sh [hunk 1]
```diff
@@ -1,52 +1,96 @@
...
+time2=$?
+
+if [ -f ${BIGDL_ROOT}/data/incd.csv ]
+then
+    echo "incd.csv already exists"
+else
+    wget -nv $FTP_URI/analytics-zoo-data/incd.csv -P ${BIGDL_ROOT}/data/
+fi
+
+execute_ray_test auto-xgboost-regressor "${BIGDL_ROOT}/python/orca/example/automl/autoxgboost/AutoXGBoostRegressor.py -p ${BIGDL_ROOT}/data/incd.csv"
```
Comment (yangw1234) [inline python/orca/dev/example/run-example-test-ray.sh:45]: some tests use `execute_ray_test`, somes tests do not. Maybe we should make them consistent.

**Comment by yangw1234** [comment]
LGTM, except one comment

**Comment by sgwhat** [comment]
The script has been consistent, new Jenkins job has also been passed.

**Pull request merged**